src/timeline.ts

```diff
@@ -42,10 +40,19 @@ class Timeline implements IAppView {
   private build() {
     // TODO build timeline using D3 of parts that doesn't change on update()
     this.$node.html(`
-      <!--<h3>${Language.TIMELINE}</h3>-->
-      <ul class="output"></ul>
-      <div id="ratioBar" class ="ratioBarChart"></div>
       <div id="timeline"></div>
+      <div id="nav-bar">
+            <button type="button" class="btn-nochanges" data-toggle="buttons-checkbox">No Changes</button>    
```

Comment:
Use a [button group](http://getbootstrap.com/components/#btn-groups) for the 4 changes buttons